wombat/src/main/include/utils/RobotStartup.h

```diff
@@ -1,25 +1,28 @@
-#pragma once 
+#pragma once
 
 #include <frc/RobotBase.h>
-#include <iostream>
+
 #include <functional>
+#include <iostream>
 namespace wom {
+namespace utils {
 
-class RobotStartup {
- public: 
-  static void Start(std::function<int()> func);
-};
+  class RobotStartup {
+  public:
+    static void Start(std::function<int()> func);
+  };
 
-template <class RobotClass>
-int StartRobot() {
-  RobotStartup::Start(frc::StartRobot<RobotClass>);
-  return 0;
-}
+  template <class RobotClass> int StartRobot() {
+    RobotStartup::Start(frc::StartRobot<RobotClass>);
+    return 0;
+  }
 
 #ifndef RUNNING_FRC_TESTS
-#define WOMBAT_ROBOT_MAIN(RobotClz) int main() { wom::StartRobot<RobotClz>(); }
-#else 
+#define WOMBAT_ROBOT_MAIN(RobotClz)                                            \
```

Comment:
random \?

Reply:
formatter lmao I I like the fact its on multiple lines so imma leave it